src/backend/funmooc/gimporter/import_scripts/helpers.py

```diff
@@ -7,17 +9,35 @@
 
 from cms.api import add_plugin, create_page
 from cms.models import Page
-from filer.models.imagemodels import Image
+from filer.models.filemodels import File as FilerFile
+from filer.models.imagemodels import Image as FilerImage
+from PIL import Image
 from pytz import timezone
 from richie.apps.courses.defaults import PAGES_INFO
 
 
-def create_image(url):
+def inflate_media(content):
```

Comment:
There is an `import_file` method just below that is called by `inflate_media`.
I renamed it to `extract_and_replace_media` because this method primarily finds the old media urls and replaces them in the content.